### DISCUSSION TITLE
why some picture not in unsorted after indexed

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#1884] [Q&A] [answered]
after index pictures in a folder，some are in Unsorted,some not, and two mp4 videos also can not be seen in Unsorted

**Comment by lastzero** [comment, marked as answer]
Did you test with the latest Development Preview? Changed the query which can make a difference.

**Reply by ghost** [reply]
I have test two version:
1. photoprism/photoprism   preview             be3f601edd36   35 hours ago    1.5GB
2. photoprism/photoprism   latest              f096af28aba5   2 weeks ago     1.5GB

both get the same result:
the picture below can be indexed and shown in Unsorted
- [pic](https://drive.google.com/file/d/1IXROWI-v-W8IAdhSrf6MbdvwEtHhBcaH/view?usp=sharing)

and the other two can be indexed and shown in Folders but not in Unsorted
- [pic1](https://drive.google.com/file/d/10Lv4P8IPn-ixtem4TxvaRyho9jmVqmox/view?usp=sharing)
- [pic2](https://drive.google.com/file/d/118kf8vu19gCfUgt5AqCWABBFN4oPXqJC/view?usp=sharing)

**Reply by graciousgrey** [reply]
That's because pic1 and pic2 are in _Review_ (https://docs.photoprism.app/user-guide/organize/review/).
If you search for `quality:0` in _Unsorted_  they are shown as well.
Alternatively you can approve them or turn off the review functionality in settings.

**Reply by ghost** [reply]
WOW！Thanks @graciousgrey

**Reply by graciousgrey** [reply]
You're welcome :)
